siriuspy/siriuspy/search/id_search.py

```diff
@@ -18,23 +22,77 @@ class IDSearch:
         'CATERETE': 'SI-07SP:ID-APU22',  # titular: VPU19
         'EMA':      'SI-08SB:ID-APU22',  # titular: IVU18
         'MANACA':   'SI-09SA:ID-APU22',  # titular: 2 x APU22
-        'SABIA':    'SI-10SB:ID-EPU50',  # titular: 2 x DELTA52
+        # 'SABIA':    'SI-10SB:ID-EPU50',  # titular: 2 x DELTA52
+        'SABIA':    'SI-10SB:ID-DELTA52',  # titular: 2 x DELTA52
         'IPE':      'SI-11SP:ID-APU58',  # titular: 2 x APPLE-II
         'PAINEIRA': 'SI-14SB:ID-WIG180',  # titular: IVU18
         'SAPUCAIA': 'SI-17SA:ID-PAPU50',  # titular: 2 x APU22
     }
 
     _idname2beamline = {v: k for k, v in _beamline2idname.items()}
 
+    _idparam_fields = (
+        'PERIOD_LENGTH',  # [mm]
+        'KPARAM_MIN',  # [mm]
+        'KPARAM_MAX',  # [mm]
+        'KPARAM_PARKED',  # [mm]
+        'KPARAM_POL_CHANGE',  # [mm]
+        'KPARAM_TOL',  # [mm]
+        'PPARAM_MIN',  # [mm]
+        'PPARAM_MAX',  # [mm]
+        'PPARAM_PARKED',  # [mm]
+        'PPARAM_TOL',  # [mm]
+        )
+
+    _idname2params = {
+        # NOTE: for EPU50 there is a large discrepancy
+        # between RB/SP/Mon phase values
+        'SI-10SB:ID-EPU50': _get_namedtuple(
+            'IDParameters',
+            _idparam_fields, (
+                50,
+                22, 300, 300, 300, 0.1,
+                -25, 25, 0, 0.5)),
+        'SI-10SB:ID-DELTA52': _get_namedtuple(
+            'IDParameters',
+            _idparam_fields, (
+                52.5,
+                -52.5/2, +52.5/2, 0, 0, 0.1,
+                -52.5/2, +52.5/2, 0, 0.1)),
+    }
+
+    POL_NONE_STR = 'none'
+    POL_UNDEF_STR = 'undef'
+
+    _idname2pol_sel = {
+        'SI-10SB:ID-EPU50': {
+            0: ('circularn', -16.36),  # [mm]
+            1: ('horizontal', 0.00),  # [mm]
+            2: ('circularp', +16.36),  # [mm]
+            3: ('vertical', 25.00),  # [mm]
+        },
+        'SI-10SB:ID-DELTA52': {  # TODO: check if values are correct
+            0: ('circularn', -16.36),  # [mm]
+            1: ('horizontal', 0.00),  # [mm]
+            2: ('circularp', +16.36),  # [mm]
+            3: ('vertical', 25.00),  # [mm]
+        },
+    }
+    _idname2pol_sts = _copy.deepcopy(_idname2pol_sel)
+    _idname2pol_sts['SI-10SB:ID-EPU50'].update(
+        {4: (POL_NONE_STR, None), 5: (POL_UNDEF_STR, None)})
+    _idname2pol_sts['SI-10SB:ID-DELTA52'].update(
+        {4: (POL_NONE_STR, None), 5: (POL_UNDEF_STR, None)})
+
     _idname_2_idff = {
         'SI-06SB:ID-APU22': None,
         'SI-07SP:ID-APU22': None,
         'SI-08SB:ID-APU22': None,
         'SI-09SA:ID-APU22': None,
         'SI-10SB:ID-EPU50': {
-            'polarizations': (
-                'none', 'circularn', 'horizontal', 'circularp', 'vertical',
-                ),
+            'polarizations': tuple(
```

Comment:
why are we still defining properties related to the EPU50?

Reply:
EPU50 might be used in the future for beamline commissioning. it is easier (for some of the developers) to have its data structure in an innocuous comment out code than having to find the appropriate commit hash number, one, two years down the road.

Reply:
but the code is not commented, is it?

Reply:
the reason I did not was so that we can you the class IDFFConfig (which depends on IDSearch) to get EPU `si_idff` configs int configdb. (there are already some configs saved in the db)